zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -97,31 +100,31 @@ impl<F: Field> ExecutionGadget<F> for CallOpGadget<F> {
             is_staticcall.expr(),
         );
         cb.condition(not::expr(is_call.expr() + is_callcode.expr()), |cb| {
-            cb.require_zero(
+            cb.require_zero_word(
                 "for non call/call code, value is zero",
-                call_gadget.value.expr(),
+                call_gadget.value.to_word(),
             );
         });
 
-        let caller_address = select::expr(
+        let caller_address = Word::select(
             is_delegatecall.expr(),
-            current_caller_address.expr(),
-            current_callee_address.expr(),
+            current_caller_address.to_word(),
+            current_callee_address.to_word(),
         );
-        let callee_address = select::expr(
+        let callee_address = Word::select(
             is_callcode.expr() + is_delegatecall.expr(),
-            current_callee_address.expr(),
-            call_gadget.callee_address_expr(),
+            current_callee_address.to_word(),
+            call_gadget.callee_address_word(),
         );
 
         // Add callee to access list
         let is_warm = cb.query_bool();
         let is_warm_prev = cb.query_bool();
-        cb.account_access_list_write(
+        cb.account_access_list_write_word(
             tx_id.expr(),
-            call_gadget.callee_address_expr(),
-            is_warm.expr(),
-            is_warm_prev.expr(),
+            call_gadget.callee_address_word(),
+            Word::from_lo_unchecked(is_warm.expr()),
+            Word::from_lo_unchecked(is_warm_prev.expr()),
```

Comment:
We can move all the `Word::from_lo_unchecked` inside the function body of `account_access_list_write_word`. 

- `is_warm` is intended to be a boolean cell.
- The conversion of `is_warm` to word is required by the RW table. 

So I think it makes sense for the circuits to be agnostic of the conversion. `account_access_list_write_word`  communicate with RW table directly and it should handle the conversion.